ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/privacy/methods/eea/EeaGetTransactionReceipt.java

```diff
@@ -101,9 +100,7 @@ public JsonRpcResponse response(final JsonRpcRequest request) {
       privateTransaction = PrivateTransaction.readFrom(bytesValueRLPInput);
       privacyGroupId = receiveResponse.getPrivacyGroupId();
     } catch (Exception e) {
-      LOG.error("Failed to fetch transaction from Enclave with error " + e.getMessage());
-      return new JsonRpcSuccessResponse(
-          request.getId(), JsonRpcError.PRIVATE_TRANSACTION_RECEIPT_ERROR);
+      return new JsonRpcSuccessResponse(request.getId(), null);
```

Comment:
_Could_ any reasons for the Exception be represented as an error? I.e. if the enclave is offline, is that a successful response? (I agree, the user probably can't do much about it ...)

Reply:
btw - "throws Exception" appears unrequired on getReceiveResponseFromEnclave

Reply:
It _could_ be represented as an error. The reason for this PR is to make the execution the same as `eth_getTransactionReceipt`, which essentially gets information from the public storage. From looking at a `RocksDbKeyValueStorage.get(...)` it throws a `StorageException` (a `RuntimeException`) which is analogous to "enclave is offline". This would return an `INTERNAL_ERROR` response.

I will make the change.